simphony/cuds/mesh.py

```diff
@@ -676,6 +684,33 @@ def has_cells(self):
         """
         return len(self._cells) > 0
 
+    def count_of(self, item_type):
+        """ Return the count of item_type in the container.
+
+        Parameter
+        ---------
+        item_type : CUBA
+            The CUBA enum of the type of the items to return the count of.
+
+        Returns
+        -------
+        count : int
+            The number of items of item_type in the container.
+
+        Raises
+        ------
+        ValueError :
+            If the type of the item is not supported in the current
+            container.
+
+        """
+
+        try:
+            return len(self._items_count[item_type]())
+        except:
```

Comment:
Maybe we should go back to:

```python
if item_type in self._items_count:
    ....
else:
    raise ValueError(msg)
```

It stills remove the double ´´´if´´´ and also,  if I explicitly capture the KeyError exceptions with the actual code I would have t re-raise them with a different message. We are not doing this anywhere else through the code.